composer.json

```diff
@@ -10,12 +10,16 @@
         "consolidation/robo": "^1.4",
         "gitonomy/gitlib": "^1.0",
         "nuvoleweb/robo-config": "^0.2.1",
-        "jakeasmith/http_build_url": "^1.0.1"
+        "jakeasmith/http_build_url": "^1.0.1",
+        "symfony/yaml": ">3.1.0"
```

Comment:
This should be checked against [Symfony's major upgrading policy](https://symfony.com/doc/current/setup/upgrade_major.html#make-your-code-deprecation-free), we should come up with a proper version string here.

Not that Drupal core is already on 3.4.* which should be deprecation free.

Something like this 

![image](https://user-images.githubusercontent.com/153362/64518862-50715000-d2f3-11e9-845a-3b6d23ef8156.png)

Reply:
What about "^3.1||4" ? 3.1 is already good enough.
Since Drupal is not the only scope for this package's usage, there is no strong reason to follow it here.